scripts/fact_detector_misuse_scope.py

```
@@ -0,0 +1,945 @@
+    """
+    if not isinstance(subject, (ast.Tuple, ast.List)):
+        return []
+    subject_elts = subject.elts
+    patterns = pattern.patterns
+    star_positions = [i for i, p in enumerate(patterns) if isinstance(p, ast.MatchStar)]
+    if len(star_positions) > 1:
+        return []
+    if not star_positions:
+        if len(patterns) != len(subject_elts):
+            return []
+        pairs = list(zip(patterns, subject_elts))
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)